src/sidebar/util/test/groups-test.js

```diff
@@ -62,16 +78,141 @@ describe('sidebar.util.groups', () => {
       const userGroups = [{ id: 'one', name: 'GroupA' }];
       const featuredGroups = [{ id: '__world__', name: 'Public' }];
 
-      const groups = combineGroups(userGroups, featuredGroups);
+      const groups = combineGroups(
+        userGroups,
+        featuredGroups,
+        'https://foo.com/bar'
+      );
       assert.equal(groups[0].id, '__world__');
     });
 
     it('handles case where there is no Public group', () => {
       const userGroups = [];
       const featuredGroups = [];
 
-      const groups = combineGroups(userGroups, featuredGroups);
+      const groups = combineGroups(
+        userGroups,
+        featuredGroups,
+        'https://foo.com/bar'
+      );
       assert.deepEqual(groups, []);
     });
+
+    it('adds isSelectable property to groups', () => {
+      const userGroups = [{ id: 'one', name: 'GroupA' }];
+      const featuredGroups = [{ id: '__world__', name: 'Public' }];
+
+      const groups = combineGroups(
+        userGroups,
+        featuredGroups,
+        'https://foo.com/bar'
+      );
+
+      groups.forEach(g => assert.equal(g.isSelectable, true));
+    });
+
+    it('sets isSelectable to true if scoped is falsey', () => {
+      const userGroups = [{ id: 'one', name: 'GroupA', scoped: false }];
+      const featuredGroups = [{ id: '__world__', name: 'Public' }];
+
+      const groups = combineGroups(
+        userGroups,
+        featuredGroups,
+        'https://foo.com/bar'
+      );
+
+      groups.forEach(g => assert.equal(g.isSelectable, true));
+    });
+
+    it('sets isSelectable to true if scopes.enforced is falsey', () => {
+      const userGroups = [
+        {
+          id: 'one',
+          name: 'GroupA',
+          scoped: true,
+          scopes: { enforced: false },
+        },
+      ];
+      const featuredGroups = [
+        { id: 'two', name: 'GroupB', scoped: true, scopes: {} },
+      ];
+
+      const groups = combineGroups(
+        userGroups,
+        featuredGroups,
+        'https://foo.com/bar'
+      );
+
+      groups.forEach(g => assert.equal(g.isSelectable, true));
+    });
+
+    it('sets isSelectable to true if scopes.uri_patterns match the uri', () => {
```

Comment:
The amount of code for each case here makes it a little tricky to see if all relevant cases are covered and add new ones. You may wish to consider combining them into a parametrized test, something along these lines:

```js
[{
  description: '<helpful description of test, useful mainly if it fails>',
  uriPatterns: [...],
  shouldBeSelectable: true | false,
}].forEach(({ description, urlPatterns, shouldBeSelectable }) => {

  it(description, () => {
    ...
    groups.forEach(g => assert.equal(g.isSelectable, shouldBeSelectable))
  });

});
```